src/Workleap.OpenApi.MSBuild/ILoggerWrapper.cs

```diff
@@ -0,0 +1,8 @@
+using Microsoft.Build.Utilities;
+
+namespace Workleap.OpenApi.MSBuild;
+
+public interface ILoggerWrapper
```

Comment:
I understand we want to create an abstraction for eventual testability but IMO it wouldn't be helpful as we would still need to create a "true" `TaskLoggingHelper` which is very difficult to mock.

```csharp
public interface ILoggerWrapper
{
    void LogWarning(string message, params object[] messageArgs);
}
```

would be testable and we could assert calls as well.